ts/src/kucoin.ts

```diff
@@ -961,19 +963,22 @@ export default class kucoin extends Exchange {
                 isTokenDepositable = isDepositEnabled || isTokenDepositable;
                 isTokenWithdrawable = isWithdrawalEnabled || isTokenWithdrawable;
                 const chainExtraData = this.safeValue (extraChainsData, chainId, {});
+                const withdrawalMinFee = this.safeNumber (chain, 'withdrawalMinFee');
+                const withdrawalMinSize = this.safeNumber (chain, 'withdrawalMinSize', 0);
+                const withdrawFeeRate = this.safeNumber (chain, 'withdrawFeeRate', 0);
                 networks[networkCode] = {
                     'info': chain,
                     'id': chainId,
                     'name': this.safeString (chain, 'chainName'),
                     'code': networkCode,
                     'active': isWithdrawalEnabled && isDepositEnabled,
-                    'fee': this.safeNumber (chain, 'withdrawalMinFee'),
+                    'fee': withdrawalMinFee + withdrawalMinSize * withdrawFeeRate,
```

Comment:
I don't think it's correct, we'd need to multiply the withdrawn amount by the fee, not the minimum size